astropy/nddata/tests/test_blocks.py

```diff
@@ -0,0 +1,164 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import numpy as np
+import pytest
+
+from astropy.nddata import reshape_as_blocks, block_reduce, block_replicate
+
+
+class TestReshapeAsBlocks:
```

Comment:
Add a test to check that `reshaped` is really a view and not a copy?

Reply:
Done.